package.json

```diff
@@ -12,6 +12,9 @@
     "node": "14.21.3",
     "yarn": "1.22.19"
   },
+  "engines": {
+    "node": ">=19"
```

Comment:
@PaulAsjes 19 is correct right? Relating to fetch + web-crypto availability.

Reply:
According to MDN, Web Crypto was fully [implemented in v19](https://developer.mozilla.org/en-US/docs/Web/API/Web_Crypto_API) and fetch was [implemented in v18](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API), so requiring at least v19 makes sense to me.